addons/scopes/stringtable.xml

```diff
@@ -4,78 +4,97 @@
         <Key ID="STR_ACE_Scopes_DisplayName">
             <English>Scopes</English>
             <Japanese>スコープ</Japanese>
+            <Polish>Celowniki optyczne</Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_enabled_displayName">
             <English>Enable ACE Scope adjustment</English>
             <Japanese>ACE スコープ調節を有効化</Japanese>
+            <Polish>Włącz ustawienia celowników optycznych ACE</Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_enabled_description">
             <English>Enable adjustmet turrets on high powered scopes</English>
             <Japanese>高倍率スコープでACE スコープ調節を有効化</Japanese>
+            <Polish>Włącz pokrętła regulacyjne </Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_forceUseOfAdjustmentTurrets_displayName">
             <English>Force adjustment turrets</English>
             <Japanese>ACE スコープ調節を有効化</Japanese>
+            <Polish>Wymus uzycie poręteł regulacyjnych</Polish>
         </Key>
         <Key ID="STR_ACE_Scopes_forceUseOfAdjustmentTurrets_description">
             <English>Force usage of adjustmet turrets on high powered scopes</English>
             <Japanese>高倍率スコープで調整の使用を強制させます</Japanese>
+            <Polish>Wymuś użycie pokręteł regulacyjnych dla celowników o duzym powiększeniu</Polish>
```

Comment:
Yea, I was thinking about using _celowniki optyczne_ for low powered sights and _lunety optyczne_ for high powered scopes.